pandas/_libs/groupby.pyx

```diff
@@ -1197,6 +1199,12 @@ cdef group_min_max(groupby_t[:, ::1] out,
         True if `values` contains datetime-like entries.
     compute_max : bint, default True
         True to compute group-wise max, False to compute min
+    mask : ndarray[bool, ndim=2], optional
```

Comment:
are we moving towards a place where we can just have one of these?

Reply:
one of what?  you mean mask vs mask_out?  it couldn't find a way to implement this without both

Reply:
ahh i understand now. ok i think then rename `mask` -> `mask_in` (i really don't like this but its less confusing i think).

Reply:
updated + green